src/components/PdfAnnotator.js

```diff
@@ -531,6 +533,7 @@ class PdfAnnotator<T_HT: T_Highlight> extends Component<
     return (
       <div
         ref={node => (this.containerNode = node)}
+        onMouseUp={this.onMouseUp}
```

Comment:
Does this work on touch devices?

Reply:
Checked and not really. Just hit a brick wall. I'll try to find a solution later

Reply:
Hi, Touch seems to be working on android but not on IOS  devices.